examples/translation-job-items.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "type": "article",
+        "id": "051273",
+        "field": "character-fr", "_field_regex": "(name|description|fits_with|varietals|character|temperature|award|storage)-(fr|it)(-[0-9]{1,4}){0,1}",
+        "name": "Mandorla Primitivo IGT Puglia"
+    },
+    {
+        "type": "online-publication-article",
+        "id": "7abb0a2d-6ff3-4a8f-8507-25bbbc168262",
+        "field": "description-fr", "_field_regex": "(name|description|fits_with|varietals|character|temperature|award|storage)-(fr|it)(-[0-9]{1,4}){0,1}",
+        "name": "Wochenaktionen (KW 46/2016): Feldschlösschen Premium Bier"
+    },
+    {
+        "type": "print-publication-article",
+        "id": "55d30cfb-54d5-41f4-98a0-69662c844cee",
+        "field": "description-fr", "_field_regex": "(name|description|fits_with|varietals|character|temperature|award|storage)-(fr|it)(-[0-9]{1,4}){0,1}",
+        "name": "Denner-Woche 2017-17: Feldschlösschen Premium Bier"
+    },
+    {
+        "type": "print-publication-sheet",
+        "id": "55d30cfb-54d5-41f4-98a0-69662c844cee",
+        "field": "footer-fr", "_field_regex": "[_a-z]+-(fr|it)",
+        "name": "Denner-Woche 2017-17: Footer Seite 1"
+    },
+    {
+        "type": "appraisal",
+        "id": "55d30cfb-54d5-41f4-98a0-69662c844cee",
+        "field": "description-fr", "_field_regex": "(title|description|response)-(fr|it)",
+        "name": "Mandorla Primitivo IGT Puglia (051273)"
+    }
+]
```

Comment:
proposal for "appraisals" has failed 😄 ...

... have thought about it, but none come to mind, I've added types

-  composite-advertised-article (thanks for suggestion)
-  newseltter-section (thanks for suggestion, but there is not much translatable text on our side - for the moment) 
-  mobile-banner